db/schema.rb

```diff
@@ -98,17 +98,20 @@
   add_index "dwp_checks", ["office_id"], name: "index_dwp_checks_on_office_id", using: :btree
 
   create_table "evidence_checks", force: :cascade do |t|
-    t.integer  "application_id", null: false
+    t.integer  "application_id",  null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.datetime "expires_at",     null: false
+    t.datetime "expires_at",      null: false
     t.boolean  "correct"
     t.integer  "income"
     t.string   "outcome"
     t.integer  "amount_to_pay"
+    t.datetime "completed_at"
+    t.integer  "completed_by_id"
```

Comment:
We briefly caught up with @dotemacs about this, because it seems unusual :) 

You're right, that it's a reference, so `_id` seems appropriate, but I haven't seen it used like this. I think the convention is usually just to use `_by`. `completed_by` is not a model anyway, so there needs to be a manual relationship setup in the `EvidenceCheck`. 

What do you think?